frame/request/batch.go

```diff
@@ -51,7 +51,7 @@ type BatchQuery struct {
 	Values   []frame.Value
 }
 
-func (q BatchQuery) WriteTo(b *frame.Buffer, name bool) { // nolint:gocritic
+func (q *BatchQuery) WriteTo(b *frame.Buffer, name bool) { // nolint:gocritic
```

Comment:
Passing by pointer should resolve gocritic issue (#71) as well, so nolint comments can be deleted.

Reply:
Great! Will change it right away, thx!

Reply:
Actually there is one more gocritic nolint in buffer.go so don't link this issue until this is fixed.